app/src/main/res/drawable/toast_frame.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+  <!-- background is tinted programmatically -->
+  <solid android:color="@android:color/holo_red_light" />
```

Comment:
This should always be tinted - if the toast looks red, something is not correct 😅